package.json

```diff
@@ -135,7 +135,8 @@
 		"vscode-languageclient": "^8.1.0"
 	},
 	"vsce": {
-		"dependencies": false
+		"dependencies": false,
+		"skipLicense": true
```

Comment:
This is required to package the extension if we do not have a `LICENSE` file. Since we're dual-licensing and have `LICENSE-MIT` and `LICENSE-APACHE`, `vsce` is not able to detect them automatically. Both licenses are packaged within the extension , though.